pagopa/io-app
# chore(IT Wallet): [SIW-1465] Fix header in eID preview screen

## Short description
This PR adds the correct close button in the eID preview screen header

## List of changes proposed in this pull request
- Added `ForceScrollDownView` in the `ItwIssuanceEidPreviewScreen`
- Manually added `HeaderSecondLevel` component via the `navigation.setOption` function

## How to test
Start the eID issuance flow. In the eID preview screen, check that the close button is correctly displayed and, when it is pressed, it shows the dismiss dialog

## Preview

<img src="https://github.com/user-attachments/assets/105c165a-029b-4a9e-9781-a17bf5ba3dbd" width="250" />
